### PR TITLE
RD-3733: Fix not closing information about the form errors

### PR DESCRIPTION
**Pull request by NorbertAmartus**
## Description

Changes consists of replacing callback for dissmissing the errors, so that the may be cleared (and with that, the information displaying the errors)

## Screenshots / Videos

CPT2112131527-1140x768

## Checklist

- [x] My code follows the UI Code Style.
- [x] I verified that all tests and checks have passed.
- [x] I verified if that change requires a documentation update.
- [x] I added proper labels to this PR.

## Tests

 

[System tests](https://jenkins.cloudify.co/view/UI%20Health%20View/job/Stage-UI-System-Test/1660) has been executed 

## Documentation

N/A

### CODE REVIEW

File: widgets/common/src/UploadBlueprintModal.tsx [hunk 1]
```diff
@@ -81,7 +99,8 @@ function UploadBlueprintModal({ toolbox, open, onHide }) {
             .doUpload(blueprintName, {
                 blueprintYamlFile,
                 blueprintUrl,
-                file: blueprintFile,
+                // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+                file: blueprintFile!,
```
Comment (NorbertAmartus) [inline widgets/common/src/UploadBlueprintModal.tsx:103]: NOTE:
At first the `blueprintFile` is declared as null (that value is being stored by typescript), but above we have multiple validations which make sure, that the `blueprintFile` is not being it's initial value 😁

File: package.json [hunk 2]
```diff
@@ -212,7 +212,7 @@
     "webpack-dev-server": "^3.11.0"
   },
   "engines": {
-    "node": "12.x"
+    "node": "14.x"
```
Comment (kubama) [inline package.json:215]: Why is this change needed?

**Review by kubama — APPROVED** [review]
Looks good, I appreciate applying "ongoing refactors".

**Review by qooban — COMMENTED** [review]
I appreciate refactors you did 👍🏻

Added minor comments. Did not test UI.

File: package.json (hunk 2 above)
Comment (NorbertAmartus) [reply package.json:215]: That change is related to the fact, that we are currently supporting version `14` of the node, as [stated here](https://github.com/cloudify-cosmo/cloudify-stage/blob/master/README.md#requirements)   

Another [package.json](https://github.com/cloudify-cosmo/cloudify-stage/blob/master/backend/package.json) is already having this kind of declaration

File: test/cypress/integration/widgets/blueprints_spec.ts [hunk 3]
```diff
@@ -343,6 +343,21 @@ describe('Blueprints widget', () => {
             cy.contains('Cancel').click();
         });
 
+        it('should successfully dismiss error messages', () => {
+            const errorBoxSeletor = '.error.message';
```
[the remark this answers is not in the document]
Comment (NorbertAmartus) [reply test/cypress/integration/widgets/blueprints_spec.ts:347]: Thanks for having an eagle eye 😁 
Fixed 😎

File: widgets/common/src/UploadBlueprintModal.tsx [hunk 4]
```diff
@@ -140,22 +161,23 @@ function UploadBlueprintModal({ toolbox, open, onHide }) {
...
-
-UploadBlueprintModal.propTypes = {
-    open: PropTypes.bool.isRequired,
-    onHide: PropTypes.func.isRequired,
-    toolbox: Stage.PropTypes.Toolbox.isRequired
 };
 
+declare global {
+    namespace Stage.Common {
+        // eslint-disable-next-line import/prefer-default-export
```
[the remark this answers is not in the document]
Comment (NorbertAmartus) [reply widgets/common/src/UploadBlueprintModal.tsx:176]: Nice one, I'll do it right now 😉

File: widgets/common/src/UploadBlueprintModal.tsx [hunk 5]
```diff
@@ -1,14 +1,32 @@
-// @ts-nocheck File not migrated fully to TS
+import { FunctionComponent } from 'react';
+
 const { BlueprintActions } = Stage.Common;
-const i18nPrefix = 'widgets.common.blueprintUpload';
+const t = Stage.Utils.getT('widgets.common.blueprintUpload');
 
 // NOTE: prevents leaking variables as global in TypeScript
 export {};
```
[the remark this answers is not in the document]
Comment (NorbertAmartus) [reply widgets/common/src/UploadBlueprintModal.tsx:7]: Oh okay, I wasn't sure about the necessity of this little hack, that's why I didn't touch it 😁 
But if it is redundant, then I'll export `UploadBlueprintModal` and remove it 😎

File: widgets/common/src/UploadBlueprintModal.tsx (hunk 4 above)
[the remark this answers is not in the document]
Comment (NorbertAmartus) [reply widgets/common/src/UploadBlueprintModal.tsx]: Done 😎

File: widgets/common/src/UploadBlueprintModal.tsx (hunk 5 above)
[the remark this answers is not in the document]
Comment (NorbertAmartus) [reply widgets/common/src/UploadBlueprintModal.tsx]: Done 🎉

**Review by qooban — APPROVED** [review]
Code looks good 👍🏻 Tested UI - works good as well! 🙂 

Thanks for addressing all the comments.
